src/app/collective-rewards/shared/components/JustifyBetweenLayout.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from 'react'
```

Comment:
Im not sure about this component yet :), 

I removed the `justify-end` since it was not necessary.

Reply:
I like this name more than the other 👍🏼